cypress/cypress.e2e.json

```diff
@@ -16,6 +16,7 @@
   "viewportHeight": 1080,
   "env": {
     "REST_URL": "https://event-bridge-event-bridge-prod.apps.openbridge-dev.fdvn.p1.openshiftapps.com",
-    "REST_PATH": "/api/smartevents_mgmt/v1"
+    "REST_PATH": "/api/smartevents_mgmt/v1",
+    "TEST_TYPE": "dev"
```

Comment:
@jomarko @ChughShilpa,
Config files are different only in few options as `"env": {"TEST_TYPE": "mocked"}` and `"baseUrl": "http://localhost:8080" }`
The rest of options is same. Do you prefer using of two separate config files or format like this https://docs.cypress.io/guides/references/configuration#Testing-Type-Specific-Overrides? Can I upgrage cypress in this JIRA?

Reply:
FYI - The migration guide: https://docs.cypress.io/guides/references/migration-guide

I found out that it reflects new changes in JavaScript.

Reply:
FYI - The old post on Cypress blog (timestamp 2 years ago - about v[4.9.x](https://docs.cypress.io/guides/references/changelog#4-9-0)): https://www.cypress.io/blog/2020/06/18/extending-the-cypress-config-file/
```Cypress does not support extends syntax in its [configuration file](https://on.cypress.io/configuration). If you want to apply different settings, you need to write a complete second configuration file and use it via the --config-file <filename> command line argument.```